test-apps/appui-test-app/standalone/src/frontend/appui/frontstages/TestPopoutFrontstage.tsx

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import "./TestPopoutFrontstage.scss";
+import * as React from "react";
+import { Frontstage } from "@itwin/appui-react";
+import { createTestFrontstage } from "./createTestFrontstage";
+
+export const createTestPopoutFrontstage = () => {
+  {
+    const frontstage = createTestFrontstage({
+      id: "appui-test-app:TestPopout",
+    });
+
+    return {
+      ...frontstage,
+      leftPanel: {
+        sections: {
+          start: [
+            {
+              id: "widget-1",
+              label: "Widget 1",
+              canPopout: true,
+              content: (
+                <>
+                  <div>Widget 1 content</div>
+                  <div id="border-test" />
+                </>
+              ),
+            },
+          ],
+        },
+      },
+    } satisfies Frontstage;
+  }
+};
+
+(() => {
+  const sheet = new CSSStyleSheet();
+  sheet.replaceSync(`
+    #border-test {
+      --border-top-color: yellow;
+      border-top: 5px solid var(--border-top-color);
+      border-right: 5px solid green;
```

Comment:
Did you mean to use a `border` shorthand here? Because otherwise the problem with `cssText` will not be seen.

Reply:
Ah, good catch, that's why it serialized correctly :) I'll need a separate element.

Reply:
Confirmed, I've just left a comment here and in `CopyStyles`.